### DISCUSSION TITLE
发现 mpv.conf 的一些参数注释上写的默认值和官网文档上面的不一样

### DISCUSSION THREAD

**Discussion by wathper** [dyphire/mpv-config#55] [General]
发现 mpv.conf 的一些参数注释上写的默认值和官网文档上面的不一样
例如：

#vd-lavc-dr=no                        # [当 gpu-api=d3d11 时] 启用直接渲染（默认：yes）。如果设置为 yes，视频将直接解码到 GPU 视频内存（或暂存缓冲区）
官网 default: auto

是 mpv.conf 中的注释过时了没有更新，还是 mpv.exe 是特殊的版本？
默认参数是以mpv.conf中的注释还是以官网的为准呢？

**Comment by wathper** [comment]
用`.\mpv --list-options`检查了一下 官方推荐的shinchiro的最新版本，和 `mpv_config-2024.03.04`的默认参数是一致的，那我就以官方文档为准了？

**Comment by dyphire** [comment]
以官方 master 文档为准，这种不常用的选项参数变动不可能一直关注，不现实

**Comment by dyphire** [comment]
在 https://github.com/dyphire/mpv-config/commit/2971a465d299ec925eb97a60cbb6fedf030036c7 提交中修正。如果发现了其他不一致的描述欢迎反馈

**Reply by wathper** [reply]
好的。
